src/main/java/hudson/plugins/git/GitChangeLogParser.java

```diff
@@ -83,7 +85,7 @@ public List<GitChangeSet> parse(@NonNull List<String> changelog) {
     @Override public GitChangeSetList parse(Run build, RepositoryBrowser<?> browser, File changelogFile)
         throws IOException, SAXException {
         // Parse the log file into GitChangeSet items - each one is a commit
-        try (Stream<String> lineStream = Files.lines(changelogFile.toPath(), StandardCharsets.UTF_8)) {
+        try (Stream<String> lineStream = Files.lines(Util.toPath(changelogFile), StandardCharsets.UTF_8)) {
```

Comment:
It was a bit risky to use `changelogFile.toPath()` since this can throw `InvalidPathException` which is not assignable to `IOException`, and who knows if some caller is depending on this to always throw `IOException`. `Util.toPath` rethrows `InvalidPathException` as `IOException`. Probably unnecessary, but better safe than sorry and I am being a bit paranoid since this is a widely installed plugin.